docs/user-guide/glossary.md

```diff
@@ -50,8 +61,13 @@ model
 example
 counterexample
   The terms "model", "example", and "counterexample" are used interchangeably.
-  They all refer to an assignment of values to all of the CVL variables and
-  contract storage.  See {ref}`rule-overview`.
+  In the context of a CVL rule, they refer to an assignment of values to all of 
+  the CVL variables and contract storage that either violates an `assert` statement 
+  or fulfills a `satisfy` statement. See {ref}`rule-overview`.
```

Comment:
for satisfy we use the term "witness example"

Reply:
ah, nice .. tweaking this entry and the `model`/`example/.. one